WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -441,20 +443,21 @@ protected void onCreate(Bundle savedInstanceState) {
 
                 if (mEditPostRepository.hasPost()) {
                     if (extras.getBoolean(EXTRA_LOAD_AUTO_SAVE_REVISION)) {
-                        mEditPostRepository.updateInTransaction(postModel -> {
-                            postModel.setTitle(
-                                    TextUtils.isEmpty(postModel.getAutoSaveTitle()) ? postModel
-                                            .getTitle()
-                                            : postModel.getAutoSaveTitle());
-                            postModel.setContent(
-                                    TextUtils.isEmpty(postModel.getAutoSaveContent()) ? postModel
-                                            .getContent()
-                                            : postModel.getAutoSaveContent());
-                            postModel.setExcerpt(
-                                    TextUtils.isEmpty(postModel.getAutoSaveExcerpt()) ? postModel
-                                            .getExcerpt()
-                                            : postModel.getAutoSaveExcerpt());
-                            return true;
+                        Log.d("vojta", "org.wordpress.android.ui.posts.EditPostActivity.onCreate");
```

Comment:
@planarvoid I just saw a few of these logs while glancing through the code in admiration 😉So pinged you just in case 😄

Reply:
good catch! I forgot to remove them

Reply:
Okay awesome🙏🏾